external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaContinuousStream.scala

```diff
@@ -184,9 +188,10 @@ class KafkaContinuousPartitionReader(
     startOffset: Long,
     kafkaParams: ju.Map[String, Object],
     pollTimeoutMs: Long,
-    failOnDataLoss: Boolean) extends ContinuousPartitionReader[InternalRow] {
+    failOnDataLoss: Boolean,
+    includeHeaders: Boolean) extends ContinuousPartitionReader[InternalRow] {
   private val consumer = KafkaDataConsumer.acquire(topicPartition, kafkaParams, useCache = false)
-  private val converter = new KafkaRecordToUnsafeRowConverter
```

Comment:
I'd have `KafkaRecordToRowProjector` (either class or object, but object would be fine) instead and move every projectors newly added in KafkaOffsetReader to there.

Reply:
So +1 on your proposal. The proposed name is just 2 cents, and I'm not sure which name fits best.

Reply:
@HeartSaVioR Great. Let's continue on discussing which name would be the best.